lib/api/BackbeatAPI.js

```diff
@@ -4,11 +4,17 @@ const async = require('async');
 const zookeeper = require('node-zookeeper-client');
 
 const { errors } = require('arsenal');
+const { RedisClient, StatsClient } = require('arsenal').metrics;
 
 const BackbeatProducer = require('../BackbeatProducer');
 const Healthcheck = require('./Healthcheck');
 const routes = require('./routes');
 
+// StatsClient constant defaults
+// TODO: This should be moved to constants file
+const INTERVAL = 300; // 5 minutes
+const EXPIRY = 900; // 15 minutes
```

Comment:
It would be good to have those two values configurable through config.json, but let's stick with this for now.